package.json

```diff
@@ -40,10 +40,14 @@
     "@types/react-dom": "^17.0.1",
     "@types/react-router-dom": "^5.1.5",
     "@types/styled-components": "^5.1.7",
+    "@web3-react/core": "^6.1.9",
+    "@web3-react/injected-connector": "^6.0.7",
+    "@web3-react/walletconnect-connector": "^6.1.9",
     "bignumber.js": "^9.0.0",
     "canvas-confetti": "^1.3.3",
     "chart.js": "^2.9.4",
     "date-fns": "^2.17.0",
+    "ethers": "^5.0.31",
```

Comment:
The package is already used here https://github.com/pancakeswap/pancake-frontend/blob/develop/src/hooks/useApprove.ts#L4 but wasn't in the deps list.
We should remove it but it's probably out of the scope of this pr